.github/labeller.yml

```diff
@@ -0,0 +1,52 @@
+labels:
+  - name: "area-ciapp"
+    title: "^\\[?(?i)ciapp"
+
+  - name: "area-serverless"
+    title: "^\\[?(?i)serverless"
+
+  - name: "area-appsec"
+    title: "^\\[?(?i)appsec"
+
+  - name: "area-profiler"
+    title: "^\\[?(?i)profiler"
+
+  - name: "area-debugger"
+    title: "^\\[?(?i)debugger"
+
+  - name: "area-profiler"
+    allFilesIn: "profiler\/.*"
+
+  - name: "area-debugger"
+    allFilesIn: "debugger\/.*"
+
+  - name: "area:builds"
+    allFilesIn: "\\.github\/.*|tracer\/build\/.*|\\.azure-pipelines\/.*|gitlab-ci\\.yml"
+
+  - name: "area:vendors"
+    allFilesIn: "tracer\/src\/Datadog\\.Trace\/vendors\/.*"
+
+  - name: "area:docs"
+    allFilesIn: "docs\/.*|.*README.*|.*\\.md"
+
+  - name: "area:installers"
+    allFilesIn: "shared\/src\/msi-installer\/.*"
+
+  - name: "area:integrations"
+    allFilesIn: "tracer\/src\/Datadog.Trace\/ClrProfiler\/.*|tracer\/src\/Datadog\\.Trace\\.AspNet\/.*|tracer\/test\/Datadog\\.Trace\\.ClrProfiler\\.IntegrationTests\/.*|tracer\/test\/Datadog\\.Trace\\.ClrProfiler\\.Managed\\.Tests\/.*"
```

Comment:
Not sure about this one exactly. ClrProfiler (for example) contains stuff not specific to "integrations" per-se, and is more of a generalised "tracer" folder, but _sometimes_ contains stuff related to integrations. I would probably limit it to Datadog.Trace.ClrProfiler.AutoInstrumentation.

Also, some of the `.` aren't escaped, not that it _really_ matters as it will still match the char `.` 😉

Reply:
Indeed, I think it's better too. Fixed it in e80d75615202e9d91f1a3c193729d9de4e17a91b